csharp/Adapter/Microsoft.Spark.CSharp/Proxy/Ipc/StreamingContextIpcProxy.cs

```diff
@@ -306,7 +305,8 @@ private void ProcessCallbackRequest(object socket)
                                 logger.LogDebug("receive close cmd from Scala side");
                                 break;
                             }
-                            else if (cmd == "callback")
```

Comment:
else if (cmd == "callback") is redundant code. I got the suggestion from ReSharp tool.